core/node/rules/can_add_event.go

```diff
@@ -815,6 +858,24 @@ func (ru *aeReceivedBlockchainTransactionRules) parentEventForReceivedBlockchain
 			StreamId: streamId,
 			Tags:     ru.params.parsedEvent.Event.Tags, // forward tags
 		}, nil
+	case *BlockchainTransaction_Transfer_:
+		if content.Transfer.GetChannelId() == nil {
+			return nil, RiverError(Err_INVALID_ARGUMENT, "transaction channel id is nil")
+		}
+		// convert to stream id
+		streamId, err := shared.StreamIdFromBytes(content.Transfer.GetChannelId())
+		if err != nil {
+			return nil, err
+		}
+		// forward the transfer to the stream as a member event, preserving the original sender as the from address
+		return &DerivedEvent{
+			Payload: events.Make_MemberPayload_BlockchainTransaction(
+				ru.receivedTransaction.FromUserAddress,
+				transaction,
+			),
+			StreamId: streamId,
+			Tags:     ru.params.parsedEvent.Event.Tags, // forward tags
+		}, nil
```

Comment:
This can just error out for now. "Received Transaction" is meant for multi party transactions like tips.